maskfile.md

```diff
@@ -27,6 +27,26 @@ else
 fi
 ~~~
 
+~~~powershell
+param (
+    $maskfile_command = $env:maskfile_command,
+    $watch = $env:watch,
+    $w = $env:w
+)
+
+if ($w) {
+    $watch = $w
+}
+
+$cargo_cmd = "cargo run -- $maskfile_command"
+$extra_args = "--exts rs --restart $cargo_cmd"
+
+if ($watch) {
+    Start-Process watchexec -ArgumentList $extra_args -NoNewWindow -PassThru
+} else {
+    cargo run -- $maskfile_command
+}
+~~~
```

Comment:
The only trouble I see in adding powershell scripts to mask's development workflow cmds is that I won't be able to maintain these easily :(. My fear is that they'll get out of date and cause issues for people trying to work on mask on Windows.

Thoughts on this? 

I could attempt to update these scripts as needed and _hope_ someone will file issues if they are ever broken. Adding a nice disclaimer to the README I could do after this merges.

Reply:
You could specify that they are in a technical alpha state of sorts.  I don't mind maintaining the scripts for as long as we use Mask.